src/libecalc/dto/result/tabular_time_series.py

```diff
@@ -110,6 +111,12 @@ def resample(self, freq: Frequency) -> Self:
                 else:
                     # NOTE: Operational settings are not resampled. Should add support?
                     pass
+
+            # Resampling compressor stream results must be done separately:
```

Comment:
Why? Since it can't inherit from TabularTimeSeries because it is missing timesteps?

Reply:
I guess my comment is a bit misleading. The comment can be replaced by: "Resampling of compressor stream results happens here". Did it clarify or was it something more you pointed at?

Reply:
Removed, added extra check further up.